README.md

```diff
@@ -1,28 +1,143 @@
-Key features:
-* promotions 
-* dynamic content management for storefront
-* content publishing management
 
-![Marketing UI](https://cloud.githubusercontent.com/assets/5801549/15642380/9767eaae-264f-11e6-8d63-e2d53129646d.png)
+**Key features**:
+
+* Dynamic content management for storefront
+* Content publishing management
+* Promotions
+
+## Dynamic Content
+
+The Dynamic Content feature allows marketers to create personalized shopping experiences for customers. Using this feature, the marketer can deliver elements of a shopping experience based on a set of conditions or context.
+
+The following should be considered before determining under which conditions the Dynamic Content will be displayed to the customer:
+
+* What type of content should be displayed;
+* The target audience who will see the content;
+* Time period when the content should be displayed;
+* Where should the content be displayed.
+
+The Dynamic Content is configured, using the Virto Commerce platform manager so that the marketing personnel can set up Dynamic Content without relying on developers and server administrators. For images and Flash animations, they may need a graphic designer to create the file and upload it to the Assets. However, once that's done, the marketer can configure rules for displaying the content without any assistance from the technical team side.
+
+The Dynamic Content consists of four independent components:
+
+* **Dynamic Content**
+* **Content Publishing**
+* **Content Type**
+* **Content Placeholder**
+
+The first two components, **Dynamic Content** and **Content Publishing**, are set up via the Virto Commerce platform manager by a marketer. The Dynamic Content provides a name and a description of the type of content that will be displayed. The Content Publishing object specifies when, under what conditions and in which placeholder the Dynamic Content should be displayed.
+
+The **Content Placeholder** is also created via VirtoCommerce platform manager, but they are created by developers, web designers and system management personnel. As soon as the  web developer adds the Content Place to a template, the developer or another user with appropriate permissions must register it in VirtoCommerce platform manager.
+
+The **Content type** is a template used to define the Dynamic Content Type. It describes how a particular content type is displayed and what information is required for it to be displayed. VirtoCommerce includes several Content types for the most common types of Dynamic Content. For example, there is a Content type to display images with links (banner) another to display product data (product with image and price).
+
+The following Dynamic **Content Types** are shipped with Virto Commerce out-of-the-box. Each of them requires one or more parameter values to specify things such as, what image file should be displayed or what the target web page should display after clicking on it.
```

Comment:
Not clear what you mean

Reply:
1. I think '-' should be written instead of ',' after 'such as'
2. clicking on it - seems 'it' in the context 'target web page', where as it is meant to be the element which we clicked.